tsconfig.json

```diff
@@ -1,12 +1,11 @@
 {
   "compilerOptions": {
-    "target": "es2015",
+    "target": "ES5",
     "module": "commonjs",
```

Comment:
hmm the rollup config has the typescript plugin.. Admitted I'm not exactly sure what it does or if they needs to be synced..

Reply:
actually nvm, I think this is correct, the rollup already uses `ES5` for target.

Reply:
yeah, it was somewhat unclear how the two of those interacted 😂 . But at least the tsconfig causes `Object.values` to error out so that's a win 🙌